packages/frontend/core/src/layouts/workspace-layout.tsx

```diff
@@ -119,6 +121,29 @@ type WorkspaceLayoutProps = {
   incompatible?: boolean;
 };
 
+// fix https://github.com/toeverything/AFFiNE/issues/4825
+function useLoadWorkspacePages() {
+  const [currentWorkspace] = useCurrentWorkspace();
+  const pageMetas = useBlockSuitePageMeta(currentWorkspace.blockSuiteWorkspace);
+  useEffect(() => {
+    if (currentWorkspace) {
+      const timer = setTimeout(() => {
+        const pageIds = pageMetas.map(meta => meta.id);
+        const pages = pageIds
+          .map(id => currentWorkspace.blockSuiteWorkspace.getPage(id))
+          .filter((p): p is Page => !!p);
+        pages.forEach(page => {
+          loadPage(page);
+        });
+      }, 10 * 1000); // load pages after 10s
```

Comment:
consider using `requestIdleCallback` to let the browser automatically defer requests?

Reply:
done